import_ldraw.py

```diff
@@ -204,7 +211,7 @@ def parse_line(self, line):
         color = line[1]
 
         if color == '16':
-            color = self.colour
+            color = self.color
```

Comment:
I am confused now. Previously you kindly forced your non-American friends to adapt to your lingo... and now you start to worry when they comply to your world dominance. :-)

No seriously, I now understand: you only wanted me to change the comments... not?